boards/wio_lite_w600/src/lib.rs

```diff
@@ -0,0 +1,334 @@
+        aliases: {
+            AlternateC: Scl
+        }
+    },
+
+    // D32/SDA/PA22
+    PA22 {
+        name: d32,
+        aliases: {
+            AlternateC: Sda
+        }
+    },
+
+    // RESET_W600
+    PA21 {
+        name: reset_w600
+    },
+
+    // D6_PA20_TCC0-W6
+    PA20 {
+        name: d6
+    },
+
+    // D12_MISO
+    PA19 {
+        name: d12
+    },
+
+    // D10_SS
+    PA18 {
+        name: d10
+    },
+
+    // D13_SCK
+    PA17 {
+        name: d13
+    },
+
+    // D11_MOSI
+    PA16 {
+        name: d11
+    },
+);
+
+/// Convenience for setting up the labelled SDA, SCL pins to
+/// operate as an I2C master running at the specified frequency.
+pub fn i2c_master<F: Into<Hertz>>(
+    clocks: &mut GenericClockController,
+    bus_speed: F,
+    sercom3: pac::SERCOM3,
+    pm: &mut pac::PM,
+    sda: Sda,
+    scl: Scl,
+) -> hal::sercom::I2CMaster3<
+    hal::sercom::Sercom3Pad0<gpio::v2::PA22>,
+    hal::sercom::Sercom3Pad1<gpio::v2::PA23>,
+> {
+    use sercom::v2::pads::Pad;
+
+    let gclk0 = clocks.gclk0();
+
+    I2CMaster3::new(
+        &clocks.sercom3_core(&gclk0).unwrap(),
+        bus_speed.into(),
+        sercom3,
+        pm,
+        Pad::new(sda),
+        Pad::new(scl),
+    )
+}
+
+/// Convenience function for setting up the D24/SCK, D23/MOSI, and D22/MISO pins
+/// as a SPI Master.
+pub fn spi_master<F: Into<Hertz>>(
+    clocks: &mut GenericClockController,
+    speed: F,
+    sercom4: pac::SERCOM4,
+    pm: &mut pac::PM,
+    sck: Sck,
+    mosi: Mosi,
+    miso: Miso,
+) -> SPIMaster4<
+    hal::sercom::Sercom4Pad0<gpio::v2::PA12>,
+    hal::sercom::Sercom4Pad2<gpio::v2::PB10>,
+    hal::sercom::Sercom4Pad3<gpio::v2::PB11>,
+> {
+    use sercom::v2::pads::Pad;
+
+    let gclk0 = clocks.gclk0();
+
+    SPIMaster4::new(
+        &clocks.sercom4_core(&gclk0).unwrap(),
+        speed.into(),
+        hal::hal::spi::Mode {
+            phase: hal::hal::spi::Phase::CaptureOnFirstTransition,
+            polarity: hal::hal::spi::Polarity::IdleLow,
+        },
+        sercom4,
+        pm,
+        (Pad::new(miso), Pad::new(mosi), Pad::new(sck)),
+    )
+}
+
+/// Convenience for setting up the D0 and D1 pins to
+/// operate as UART RX/TX (respectively) running at the specified baud.
+pub fn uart<F: Into<Hertz>>(
+    clocks: &mut GenericClockController,
+    baud: F,
+    sercom2: pac::SERCOM2,
+    pm: &mut pac::PM,
+    rx: Rx,
+    tx: Tx,
+) -> UART2<
+    hal::sercom::Sercom2Pad3<gpio::v2::PA11>,
+    hal::sercom::Sercom2Pad2<gpio::v2::PA10>,
+    (),
+    (),
+> {
+    use sercom::v2::pads::Pad;
+
+    let gclk0 = clocks.gclk0();
+
+    UART2::new(
+        &clocks.sercom2_core(&gclk0).unwrap(),
+        baud.into(),
+        sercom2,
+        pm,
+        (Pad::new(rx), Pad::new(tx)),
+    )
+}
+
+#[cfg(feature = "usb")]
+pub fn usb_allocator(
+    usb: pac::USB,
+    clocks: &mut GenericClockController,
+    pm: &mut pac::PM,
+    dm: impl AnyPin<Id=PA24>,
+    dp: impl AnyPin<Id=PA25>,
```

Comment:
For consistency, you could add USB aliases for these pins in the macro above. Then this function would look like the others.

Reply:
Ah yeah, good point. Also found these pins were labeled backwards 🤦🏻‍♂️